node_modules/oae-core/activity/js/activity.js

```diff
@@ -122,8 +122,9 @@ define(['jquery', 'underscore', 'oae.core', 'activityadapter'], function($, _, o
             infinityScroll = $('.oae-list', $rootel).infiniteScroll(url, {
                 'limit': 10
             }, '#activity-items-template', {
-                'postProcessor': processActivities,
-                'emptyListProcessor': handleEmptyResultList
+                'ariaLive': 'off',  // Notifications will alert screen readers
```

Comment:
It's a bit tricky to determine whether an activity that gets placed in the activity stream gets pushed out as a notification as well in the UI though.
IIRC, the `activityId` between an activity and a notification will be different. So you can't compare on that.

Notifications are pushed to users on aggregation, which means you'd have to aggregate incoming activities with older activities (which may or may not be available to the UI) and then wait up to 5 seconds until a notification rolls in. Other than the delay implication, this would probably work for most use-cases, but would still be a pain as you'd then have to copy over all the backend aggregation rules.

Not reading a new activity (that has no notification) doesn't seem like such a big problem to me. We've already deemed it "not worthy" of being sent to the notification stream. Maybe it's ok to let the user discover it in their activity stream on a hard page refresh?